module/solutions/helper.js

```diff
@@ -290,10 +290,28 @@ module.exports = class SolutionsHelper {
         if (getTargetedSolution) {
           targetedSolutions = await this.forUserRoleAndLocation(requestedData, solutionType, '', '', '', '', search);
         }
+        
         if (targetedSolutions.success) {
-          if (targetedSolutions.success && targetedSolutions.data.data && targetedSolutions.data.data.length > 0) {
-            totalCount += targetedSolutions.data.count;
-            targetedSolutions.data.data.forEach((targetedSolution) => {
+					// When targetedSolutions is empty and currentScopeOnly is set to true send empty response
+					if (!(targetedSolutions.data.data.length > 0) && currentScopeOnly) {
```

Comment:
@borkarsaish65 please update this function signature with description for currentScopeOnly

Reply:
done